console/src/components/Services/Data/TableRelationships/RemoteRelationships/RemoteRelationships.js

```diff
@@ -13,11 +13,11 @@ const RemoteRelationships = ({
 }) => {
   // restructure existing relationships to the state structure
   const existingRemoteRelationships = tableSchema.remote_relationships.map(
-    rr => {
+    ({ definition = {}, remote_relationship_name }) => {
       return parseRemoteRelationship({
-        remote_schema: rr.configuration.remote_schema,
-        remote_field: rr.configuration.remote_field,
-        name: rr.configuration.name,
+        remote_schema: definition?.remote_schema || 'Invalid Remote Schema!',
```

Comment:
1. `definition` shall never be null or undefined
2.  `remote_field` and `name` shall also never be null or undefined.

So the conditional access and default values are not needed.

Reply:
Okay. I will revert these changes.

Reply:
Done 👍